.changeset/stale-days-smoke.md

```diff
@@ -0,0 +1,5 @@
+---
+'@swisspost/design-system-components': major
```

Comment:
You did not remove anything therefore it's not a major change.
Also, you did not update the `@swisspost/design-system-components`.
Can you update this changeset please?